src/main/resources/hw16-screenshots/hw17.sql

```diff
@@ -0,0 +1,153 @@
+
+--Display a list of employees whose plans exceed the plans of their leaders
+select s.name, s.quota, mgrs.quota from salesreps s
+inner join salesreps mgrs on s.manager = mgrs.empl_num and s.quota > mgrs.quota;
+
+--Display a list of employees who work with their managers in various offices
+--including the names and offices of both employees and managers.
+select s.name, so.city, mgrs.name, mgrso.city from salesreps s
+inner join offices so on s.rep_office = so.office
+inner join salesreps mgrs on s.manager = mgrs.empl_num and s.rep_office <> mgrs.rep_office
+inner join offices mgrso on mgrs.rep_office = mgrso.office;
+
+--output all name of offices with their workers
+select s.name, city from offices o
+inner join salesreps s on o.office is not null;
+
+--step 31
+
+--Output averege quota and avg sales from salesreps
+select avg(quota), avg(sales) from salesreps;
+
+--Output averege plan-execution from salesreps
+select avg(100 * (sales/quota)) from salesreps;
+select avg(100 * (sales/quota)) procent from salesreps;
+
+--Output sum of quota and sales from salesreps
+select sum(quota), sum (sales) from salesreps;
+
+--Output amount of all offers recived by Dima Malkov
+select sum(amount) from salesreps s
+inner join orders o on s.name = '���� �������' and o.rep = s.empl_num;
+--Amount mean a profit, horrible naming
+
+--Output avg price of ACI manufacturer products
+select avg(price) from products where mfr_id = 'ACI';
+
+--Output avg amount of 2103 custumer's offers; 
+select avg(amount) from orders where cust = 2103;
+
+--step 32
+
+--Output minimal and maximal quota from salereps
+select min(quota), max(quota) from salesreps;
+
+--Output date of oldest order
+select min(order_date) from orders;
+
+--Output percent of plant executing of most succssful saler
+select max(100 * (sales/quota)) from salesreps;
+
+--How many clients do we have
+select count(*)from customers;
+
+--how many salers done the plan
+select count(*) from salesreps where sales > quota;
+
+--Output all quantity of orders with an amount > 250;
+select count(*) from orders where amount > 250;
+
+--Find the average cost of orders, the total cost of orders, the average cost
+--of orders as a percentage of customer loan limits,
+--as well as the average cost of orders as a percentage of the planned sales of employees.
+select avg(amount), sum(amount), (100 * avg(amount/credit_limit)), (100 * avg(amount/quota)) from orders o
+inner join customers c on o.cust = c.cust_num
+inner join salesreps s on o.rep = s.empl_num;
+    
+select amount, amount, amount/credit_limit, amount/quota from orders o
+inner join customers c on o.cust = c.cust_num
+inner join salesreps s on o.rep = s.empl_num;
+
+select distinct c.* from customers c
+inner join orders o on c.cust_num = o.cust
+inner join products p on o.product = p.product_id
+and p.description like '%A%' or p.description like '%a%' or p.description like '%�%' or p.description like '%�%';
```

Comment:
Why in the homework you add step 32-26, this is the previous homework?

Reply:
Wery good homewok!

Reply:
Oh, didnt notice. Sims like I pushed in existed branch. Sorry